packages/x-codemod/src/v6.0.0/data-grid/rename-linkOperators-logicOperators/index.ts

```diff
@@ -0,0 +1,81 @@
+import { ASTPath, ImportDeclaration } from 'jscodeshift';
+import { JsCodeShiftAPI, JsCodeShiftFileInfo } from '../../../types';
+
+const renamedIdentifiers = {
+  GridLinkOperator: 'GridLogicOperator',
+  linkOperatorInputProps: 'logicOperatorInputProps',
+  linkOperator: 'logicOperator',
+  linkOperators: 'logicOperators',
+  setFilterLinkOperator: 'setFilterLogicOperator',
+};
+
+const renamedLiterals = {
+  filterPanelLinkOperator: 'filterPanelLogicOperator',
+};
+
+const renamedClasses = {
+  'MuiDataGrid-filterFormLinkOperatorInput': 'MuiDataGrid-filterFormLogicOperatorInput',
+  'MuiDataGrid-withBorder': 'MuiDataGrid-withBorderColor',
+};
+
+const PACKAGE_REGEXP = /@mui\/x-data-grid(-pro|-premium)?/;
+
+const matchImport = (path: ASTPath<ImportDeclaration>) =>
+  ((path.node.source.value as any).toString() || '').match(PACKAGE_REGEXP);
+
+// Press ctrl+space for code completion
+export default function transform(file: JsCodeShiftFileInfo, api: JsCodeShiftAPI, options: any) {
+  const j = api.jscodeshift;
+  const root = j(file.source);
+
+  const printOptions = options.printOptions || {
+    quote: 'single',
+    trailingComma: true,
+  };
+
+  const matchingImports = root.find(j.ImportDeclaration).filter((path) => !!matchImport(path));
+
+  if (matchingImports.length > 0) {
+    // Rename the identifiers
+    // - import { linkOperatorInputProps } from '@mui/x-data-grid'
+    // + import { logicOperatorInputProps } from '@mui/x-data-grid'
```

Comment:
Am I missing something, but I don't see such case in tests, nor codemod readme diff... 🙈 🤷

Reply:
Correct, updated!